src/rust-crypto/verification.ts

```diff
@@ -659,18 +660,23 @@ export class RustSASVerifier extends BaseRustVerifer<RustSdkCryptoJs.Sas> implem
     /** if we can now show the callbacks, do so */
     protected onChange(): void {
         if (this.callbacks === null) {
-            const emoji: Array<Emoji> | undefined = this.inner.emoji();
```

Comment:
might as well clean this up while we're here: the return type of `emoji` is now `Array<Emoji> | undefined`